anoncreds/src/data_types/anoncreds/rev_reg.rs

```diff
@@ -73,14 +84,58 @@ impl RevocationStatusList {
         self.revocation_list.get(idx).as_deref().copied()
     }
 
+    pub(crate) fn update(
+        &mut self,
+        registry: Option<ursa::cl::RevocationRegistry>,
+        issued: Option<BTreeSet<u32>>,
+        revoked: Option<BTreeSet<u32>>,
+        timestamp: Option<u64>,
+    ) -> Result<(), error::Error> {
+        if let Some(reg) = registry {
+            self.registry = Some(reg)
+        };
+        if let Some(issued) = issued {
+            // issued credentials are assigned `false`
+            // i.e. NOT revoked
+            for i in issued {
+                let mut bit = self.revocation_list.get_mut(i as usize).ok_or_else(|| {
+                    error::Error::from_msg(
+                        crate::ErrorKind::Unexpected,
+                        "Update Revocation List Index Out of Range",
+                    )
+                })?;
+                *bit = false;
+            }
+        }
+        if let Some(revoked) = revoked {
+            // revoked credentials are assigned `true`
+            // i.e. IS revoked
+            for i in revoked {
+                let mut bit = self.revocation_list.get_mut(i as usize).ok_or_else(|| {
+                    error::Error::from_msg(
+                        crate::ErrorKind::Unexpected,
+                        "Update Revocation List Index Out of Range",
+                    )
+                })?;
+                *bit = true;
+            }
+        }
+        if let Some(t) = timestamp {
+            self.timestamp = Some(t);
+        }
```

Comment:
Same here.

Reply:
Same as above, added doc, happy to discuss more